lib/streamlit/delta_generator.py

```diff
@@ -574,7 +576,9 @@ def _enqueue(
 
     def _block(
         self,
-        block_proto: Block_pb2.Block = Block_pb2.Block(),
+        block_proto: Block_pb2.Block = Block_pb2.Block(
+            vertical=Block_pb2.Block.Vertical()
```

Comment:
This seems a bit risky. Are we sure that there aren't any unintended issues with this change?